src/components/Privacy/PrivacyHeader.tsx

```diff
@@ -4,17 +4,19 @@ import { usePrivacyMetadata } from '@hooks/usePrivacyMetadata'
 import PrivacyLanguages from './PrivacyLanguages'
 
 export default function PrivacyPolicyHeader({
-  policy
+  policy,
+  isCookie = false
```

Comment:
todo: refactor this component to a more generic `PolicyHeader` component which can be used for cookies, privacy and future policies.